crates/stackable-webhook/src/tls.rs

```diff
@@ -139,10 +139,24 @@ impl TlsServer {
                     socket_addr: self.socket_addr,
                 })?;
 
+        // To be able to extract the connect info from incoming requests, it is
+        // required to turn the router into a Tower service which is capable of
+        // doing that. Calling `into_make_service_with_connect_info` returns a
+        // new struct `IntoMakeServiceWithConnectInfo` which implements the
+        // Tower Service trait. This service is called after the TCP connection
+        // has been accepted.
+        //
+        // Inspired by:
+        // - https://github.com/tokio-rs/axum/discussions/2397
+        // - https://github.com/tokio-rs/axum/blob/b02ce307371a973039018a13fa012af14775948c/examples/serve-with-hyper/src/main.rs#L98
+
+        let mut router = self
```

Comment:
**question**: As I understand this, `router: IntoMakeServiceWithConnectInfo`. What was the type before?

Reply:
Before, it was just `router: Router` iirc.